packages/terser/test/inline_sourcemap/input.js

```diff
@@ -1,11 +1,6 @@
-'use strict';
-exports.__esModule = true;
 // clang-format off
-var MyClass = /** @class */ (function () {
-    function MyClass(something) {
+export class MyClass {
```

Comment:
can you explain why these changed?

Reply:
I've remembered at some point I've ran `yarn format` probably it was a result of it. I'll recover the original version